### DISCUSSION TITLE
how to display the topic in MQTTX CLI Sub

### DISCUSSION THREAD

**Discussion by rido-min** [emqx/MQTTX#1601] [Q&A] [answered]
When I use the CLI `mqttx sub -t '#'`  I only see the message payload and would like to see the topic.

I tried with clean mode, but then payload is shown as bytes.

**Comment by Red-Asuka** [comment, marked as answer]
In the next version, we adjusted the default display content of the sub command. In addition to payload, topic and qos will also be displayed. For details, see: https://github.com/emqx/MQTTX/pull/1592

Before the new version is released, you can use the `-v` parameter to display topics.  For more options and usage, see: https://mqttx.app/docs/cli/get-started#subscribe

<img width="375" alt="image" src="https://github.com/emqx/MQTTX/assets/38158783/34bdad48-0894-4099-8393-16872661e5f7">
